### PR TITLE
Set up linters

### PR DESCRIPTION
**Pull request by od-c0d3r**
1st pull req

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <header class="thicFont">
+      This is my world
```
Comment (elisha2kyakpo1) [inline index.html:10]: Please follow the instruction on the header message, should read "Hello world"

Comment (elisha2kyakpo1) [reply index.html:10]: Everything looks good except for the header message.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (od-c0d3r) [reply index.html]: Thank you, code updated

**Comment by elisha2kyakpo1** [comment]
It's good to see you putting in a lot of effort

**Comment by od-c0d3r** [comment]
Thanks to @iam-Akshat for hinting me the Proplem with yml file.

**Review by od-c0d3r — COMMENTED** [review]
all good

**Pull request merged**